demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php

```diff
@@ -2143,7 +2143,8 @@ public function administrationMemberListAction(
 
         $emailTextAdded = '';
         if ($this->permissions->hasPermission('feature_email_invitable_institution_additional_invitation_text')) {
-            $emailTextAdded = $this->generateAdditionalInvitationEmailText($publicAffairsAgents, $organization, $procedureAsArray['agencyMainEmailAddress']);
+            $agencyMainEmailAddress = isset($procedureAsArray['agencyMainEmailAddress']) && null !== $procedureAsArray['agencyMainEmailAddress'] ? $procedureAsArray['agencyMainEmailAddress'] : '';
+            $emailTextAdded = $this->generateAdditionalInvitationEmailText($publicAffairsAgents, $organization, $agencyMainEmailAddress);
```

Comment:
```suggestion
            $emailTextAdded = $this->generateAdditionalInvitationEmailText($publicAffairsAgents, $organization, $procedureAsArray['agencyMainEmailAddress'] ?? '');
```

Reply:
done in [cb1a76c](https://github.com/demos-europe/demosplan-core/pull/1645/commits/cb1a76c566ac32de04160e486908541c32a6ae84)